schemas/host.json

```diff
@@ -431,6 +431,16 @@
                       "description": "The target rate that the adaptive algorithm aims for on each instance",
                       "type": "integer",
                       "default": 5
+                    },
+                    "excludedTypes": {
+                      "description": "A semi-colon delimited list of types that should be excluded from sampling. All types not listed will be sampled.",
```

Comment:
In the description, can you list the possible types (same with below)? Types are listed here: https://docs.microsoft.com/en-us/azure/azure-monitor/app/sampling#configuring-adaptive-sampling-for-aspnet-applications

You may be able to just copy their description as-is for these...